regression-tests/mixed-type-safety-1.cpp2

```diff
@@ -13,7 +13,7 @@ class Square : public Shape { };
 print: ( msg: std::string, x: _ ) =
     std::cout << msg << x << "\n";
 
-print: ( msg: std::string, b: bool ) =
+print: <B:std::convertible_to<bool>> ( msg: std::string, copy b: B ) =
```

Comment:
FYI, what I did at <https://github.com/hsutter/cppfront/pull/529/files#diff-508db04ed7e2433d79b037f5473ae5ca5330b4b04b842136ad0c95a40142fee8>
was to remove this function and preface `main` with `std::cout << std::boolalpha;`.

Reply:
I just thought that two versions of the `print` functions were to show that the result will match the one with `bool`. If it is not needed then your version is the right one.

Reply:
We'd have to ask @hsutter for the purpose.